apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src

```diff
@@ -1,24 +1,20 @@
-%% -*-: erlang -*-
-
-{"4.3.3",
-  [
-    {<<"4.3.[1-2]">>, [
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {"4.3.0", [
-      {load_module, emqx_bridge_worker, brutal_purge, soft_purge, []},
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {<<".*">>, []}
-  ],
-  [
-    {<<"4.3.[1-2]">>, [
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {"4.3.0", [
-      {load_module, emqx_bridge_worker, brutal_purge, soft_purge, []},
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {<<".*">>, []}
-  ]
-}.
+%% -*- mode: erlang -*-
+{VSN,
```

Comment:
Here we should have correct version.

Reply:
I believe that rebar3 substitutes this `VSN` with the app version from `app.src` (there are other appup's that do the same)

Reply:
@thalesmg So what should I change and how?
I've read this doc

https://www.erlang.org/doc/man/appup.html

Reply:
Great, thanks!